Slango/Slango/HomeViewController.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  HomeViewController.swift
+//  Slango
+//
+//  Created by Fabiola S on 2/27/20.
+//  Copyright © 2020 Fabiola Saga. All rights reserved.
+//
+
+import UIKit
+
+class HomeViewController: UIViewController {
+    
+    // MARK: Outlets
+    @IBOutlet weak var backgroundGradient: UIView!
+    
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setGradientBackgroundColor()
+        // Do any additional setup after loading the view.
+    }
+    
+    // MARK: - UI
+    
+    func setGradientBackgroundColor() {
+        
+        let gradientLayer = CAGradientLayer()
+        gradientLayer.frame = view.bounds
+        gradientLayer.colors = [#colorLiteral(red: 0, green: 0.3529411765, blue: 0.737254902, alpha: 1).cgColor, #colorLiteral(red: 0, green: 0.7573584914, blue: 1, alpha: 1).cgColor ]
```

Comment:
Color literals won’t work with DarkMode. You can use asset catalog colors instead though!

Reply:
Thanks for the suggestion! Will try that right now.